astropy/utils/data.py

```diff
@@ -991,10 +997,18 @@ def _download_file_from_source(source_url, show_progress=True, timeout=None,
     if http_headers is None:
         http_headers = {}
 
+    if os.name == 'nt':
+        ssl_context = ssl.create_default_context(cafile=certifi.where())
+    else:
+        ssl_context = ssl.create_default_context()
+
+    https_handler = urllib.request.HTTPSHandler(context=ssl_context)
```

Comment:
What are this and the previous lines of code for?  It doesn't look like `https_handler` is used anywhere`.

Reply:
Nevermind, I see you changed this later.